### DISCUSSION TITLE
Outdated Python default certificate detected.

### DISCUSSION THREAD

**Discussion by Wix010** [thoughtspot/cs_tools#143] [General]
Hello,
I installed cs_tool and now I would like to create config file, but I have this issue. Suggested quick fix does not work beacaase config file has not been created.
![image](https://github.com/thoughtspot/cs_tools/assets/170716946/01c5a50a-430b-475a-9f9f-be20049cc540)

**Comment by boonhapus** [comment]
@Wix010 if you try running the `config create` command again, you can add the `--disable-ssl` flag to get the same effect.

It would be good if I checked this scenario (create vs modify) or just allowed the creation aspect to go through even if it fails the check. I'll look into adding that!

**Reply by Wix010** [reply]
No, it does not work. Finished with the same error message.

**Reply by boonhapus** [reply]
Hey @Wix010 sorry for the slow response on this one -- I've changed the message here to reflect the correct flag, as well as allowing saving of invalid config files.
